repl/src/test/scala/com/cloudera/livy/repl/BaseSessionSpec.scala

```diff
@@ -22,18 +22,28 @@ import scala.concurrent.Await
 import scala.concurrent.duration._
 import scala.language.postfixOps
 
-import org.json4s.DefaultFormats
+import org.json4s._
 import org.scalatest.{FlatSpec, Matchers}
 import org.scalatest.concurrent.Eventually._
 
 import com.cloudera.livy.LivyBaseUnitTestSuite
+import com.cloudera.livy.rsc.driver.{Statement, StatementState}
 import com.cloudera.livy.sessions.SessionState
 
 abstract class BaseSessionSpec extends FlatSpec with Matchers with LivyBaseUnitTestSuite {
 
   implicit val formats = DefaultFormats
 
-  def withSession(testCode: Session => Any): Unit = {
+  protected def execute(session: Session, code: String): Statement = {
```

Comment:
Yes, there's no big difference, what I want is to keep the same style as original code `withXXX()` or `withXXX(xxx)(...)`. I don't strongly ask you to change, just a suggestion.